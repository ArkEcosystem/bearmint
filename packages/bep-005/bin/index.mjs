#!/usr/bin/env node
import { join } from 'node:path'

import { globby } from 'globby'
import meow from 'meow'

process.env.NODE_ENV = 'test'

const { flags } = meow({
	flags: {
		config: {
			type: 'string',
		},
	},
	importMeta: import.meta,
})

const { files } = (await import(join(process.cwd(), flags.config))).default
const tests = (await globby(files)).map((path) => join(process.cwd(), path))

for (const test of tests) {
	await import(test)
}
