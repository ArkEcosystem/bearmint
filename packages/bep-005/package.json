{
	"name": "@bearmint/bep-005",
	"version": "0.0.0",
	"private": true,
	"description": "This library is the official reference implementation for BEP-005 (https://bep.dev/005)",
	"keywords": [
		"bearmint",
		"tendermint",
		"blockchain",
		"framework",
		"modular"
	],
	"homepage": "https://github.com/bearmint/bearmint",
	"repository": {
		"type": "git",
		"url": "git+https://github.com/bearmint/bearmint.git"
	},
	"funding": "https://funding.bearmint.com/",
	"license": "MIT",
	"author": "Bearmint Mining Company <hello@bearmint.com> (https://bearmint.com/)",
	"type": "module",
	"exports": "./distribution/index.js",
	"types": "./distribution/index.d.ts",
	"bin": {
		"roadrunner": "./bin/index.mjs"
	},
	"files": [
		"bin",
		"distribution"
	],
	"scripts": {
		"audit": "pnpm audit --fix --prod",
		"build": "pnpm run clean && tsc",
		"build:watch": "pnpm run clean && tsc -w",
		"clean": "rm -rf distribution",
		"depcheck": "depcheck --ignores=@bearmint/bep-004,@bearmint/bep-005,@bearmint/bep-031,@bearmint/bep-039,globby,meow",
		"deps:major": "ncu -u --target=latest",
		"deps:minor": "ncu -u --target=minor",
		"deps:patch": "ncu -u --target=patch"
	},
	"dependencies": {
		"@bearmint/bep-013": "workspace:^",
		"@bearmint/bep-018": "workspace:^",
		"@bearmint/bep-116": "workspace:^",
		"@sinonjs/fake-timers": "9.1.2",
		"ansi-styles": "6.2.1",
		"concordance": "5.0.4",
		"globby": "13.1.2",
		"lodash.get": "4.4.2",
		"lodash.has": "4.5.2",
		"meow": "^11.0.0",
		"nock": "13.2.9",
		"pretty-ms": "8.0.0",
		"sinon": "14.0.1",
		"strip-ansi": "7.0.1",
		"vitest": "^0.24.5"
	},
	"devDependencies": {
		"@types/node": "18.11.9",
		"@types/sinon": "10.0.13",
		"@types/sinonjs__fake-timers": "8.1.2"
	},
	"engines": {
		"node": ">=18",
		"pnpm": ">=7.9.0"
	},
	"publishConfig": {
		"access": "restricted"
	}
}
