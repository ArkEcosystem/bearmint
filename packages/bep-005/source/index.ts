export * from './expect.js'
export * from './fake.js'
export * from './nock.js'
export * from './spy.js'
export * from './stub.js'
