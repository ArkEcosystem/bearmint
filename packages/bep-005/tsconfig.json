{
	"compilerOptions": {
		"typeRoots": ["node_modules/@types"],
		"outDir": "distribution"
	},
	"exclude": ["node_modules", "test/**/**.ts"],
	"extends": "../../config/tsconfig.json",
	"include": ["source/**/**.ts"]
}
