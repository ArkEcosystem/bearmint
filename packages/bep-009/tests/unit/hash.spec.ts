import { expect, describe, it, beforeEach } from 'vitest'
import { extendExpect } from '@bearmint/bep-005'
extendExpect(expect)

import {
	blake2b,
	blake2s,
	blake3,
	keccak_224,
	keccak_256,
	keccak_384,
	keccak_512,
	ripemd160,
	sha1,
	sha3_224,
	sha3_256,
	sha3_384,
	sha3_512,
	sha256,
	sha384,
	sha512,
	shake128,
	shake256,
} from '../../source/hash.js'

describe('Hash', () => {
	it('blake2b', () => {
		expect(blake2b(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(blake2b('Hello, World!')).toMatchSnapshot()
	})

	it('blake2s', () => {
		expect(blake2s(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(blake2s('Hello, World!')).toMatchSnapshot()
	})

	it('blake3', () => {
		expect(blake3(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(blake3('Hello, World!')).toMatchSnapshot()
	})

	it('keccak_224', () => {
		expect(keccak_224(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(keccak_224('Hello, World!')).toMatchSnapshot()
	})

	it('keccak_256', () => {
		expect(keccak_256(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(keccak_256('Hello, World!')).toMatchSnapshot()
	})

	it('keccak_384', () => {
		expect(keccak_384(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(keccak_384('Hello, World!')).toMatchSnapshot()
	})

	it('keccak_512', () => {
		expect(keccak_512(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(keccak_512('Hello, World!')).toMatchSnapshot()
	})

	it('ripemd160', () => {
		expect(ripemd160(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(ripemd160('Hello, World!')).toMatchSnapshot()
	})

	it('sha1', () => {
		expect(sha1(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha1('Hello, World!')).toMatchSnapshot()
	})

	it('sha3_224', () => {
		expect(sha3_224(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha3_224('Hello, World!')).toMatchSnapshot()
	})

	it('sha3_256', () => {
		expect(sha3_256(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha3_256('Hello, World!')).toMatchSnapshot()
	})

	it('sha3_384', () => {
		expect(sha3_384(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha3_384('Hello, World!')).toMatchSnapshot()
	})

	it('sha3_512', () => {
		expect(sha3_512(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha3_512('Hello, World!')).toMatchSnapshot()
	})

	it('sha256', () => {
		expect(sha256(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha256('Hello, World!')).toMatchSnapshot()
	})

	it('sha384', () => {
		expect(sha384(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha384('Hello, World!')).toMatchSnapshot()
	})

	it('sha512', () => {
		expect(sha512(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(sha512('Hello, World!')).toMatchSnapshot()
	})

	it('shake128', () => {
		expect(shake128(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(shake128('Hello, World!')).toMatchSnapshot()
	})

	it('shake256', () => {
		expect(shake256(Buffer.from('Hello, World!'))).toMatchSnapshot()
		expect(shake256('Hello, World!')).toMatchSnapshot()
	})
})
