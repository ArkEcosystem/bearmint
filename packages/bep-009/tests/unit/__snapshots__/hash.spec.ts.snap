// Vitest Snapshot v1

exports[`Hash > blake2b 1`] = `
Uint8Array [
  125,
  253,
  184,
  136,
  175,
  113,
  234,
  224,
  230,
  166,
  183,
  81,
  232,
  227,
  65,
  61,
  118,
  126,
  244,
  250,
  82,
  167,
  153,
  61,
  170,
  158,
  240,
  151,
  247,
  170,
  61,
  148,
  145,
  153,
  193,
  19,
  202,
  163,
  124,
  148,
  248,
  12,
  243,
  178,
  47,
  125,
  157,
  110,
  79,
  93,
  239,
  79,
  249,
  39,
  131,
  12,
  255,
  228,
  133,
  124,
  52,
  190,
  61,
  137,
]
`;

exports[`Hash > blake2b 2`] = `
Uint8Array [
  125,
  253,
  184,
  136,
  175,
  113,
  234,
  224,
  230,
  166,
  183,
  81,
  232,
  227,
  65,
  61,
  118,
  126,
  244,
  250,
  82,
  167,
  153,
  61,
  170,
  158,
  240,
  151,
  247,
  170,
  61,
  148,
  145,
  153,
  193,
  19,
  202,
  163,
  124,
  148,
  248,
  12,
  243,
  178,
  47,
  125,
  157,
  110,
  79,
  93,
  239,
  79,
  249,
  39,
  131,
  12,
  255,
  228,
  133,
  124,
  52,
  190,
  61,
  137,
]
`;

exports[`Hash > blake2s 1`] = `
Uint8Array [
  236,
  157,
  185,
  4,
  214,
  54,
  239,
  97,
  241,
  66,
  27,
  43,
  164,
  113,
  18,
  164,
  250,
  107,
  137,
  100,
  253,
  74,
  10,
  81,
  72,
  52,
  69,
  92,
  33,
  223,
  120,
  18,
]
`;

exports[`Hash > blake2s 2`] = `
Uint8Array [
  236,
  157,
  185,
  4,
  214,
  54,
  239,
  97,
  241,
  66,
  27,
  43,
  164,
  113,
  18,
  164,
  250,
  107,
  137,
  100,
  253,
  74,
  10,
  81,
  72,
  52,
  69,
  92,
  33,
  223,
  120,
  18,
]
`;

exports[`Hash > blake3 1`] = `
Uint8Array [
  40,
  138,
  134,
  167,
  159,
  32,
  163,
  214,
  220,
  205,
  202,
  119,
  19,
  190,
  174,
  209,
  120,
  121,
  130,
  150,
  189,
  250,
  121,
  19,
  250,
  42,
  98,
  217,
  114,
  123,
  248,
  248,
]
`;

exports[`Hash > blake3 2`] = `
Uint8Array [
  40,
  138,
  134,
  167,
  159,
  32,
  163,
  214,
  220,
  205,
  202,
  119,
  19,
  190,
  174,
  209,
  120,
  121,
  130,
  150,
  189,
  250,
  121,
  19,
  250,
  42,
  98,
  217,
  114,
  123,
  248,
  248,
]
`;

exports[`Hash > keccak_224 1`] = `
Uint8Array [
  78,
  170,
  240,
  231,
  161,
  228,
  0,
  239,
  186,
  113,
  19,
  7,
  34,
  225,
  203,
  77,
  89,
  179,
  42,
  251,
  64,
  14,
  101,
  74,
  254,
  196,
  248,
  206,
]
`;

exports[`Hash > keccak_224 2`] = `
Uint8Array [
  78,
  170,
  240,
  231,
  161,
  228,
  0,
  239,
  186,
  113,
  19,
  7,
  34,
  225,
  203,
  77,
  89,
  179,
  42,
  251,
  64,
  14,
  101,
  74,
  254,
  196,
  248,
  206,
]
`;

exports[`Hash > keccak_256 1`] = `
Uint8Array [
  172,
  175,
  50,
  137,
  215,
  182,
  1,
  203,
  209,
  20,
  251,
  54,
  196,
  210,
  156,
  133,
  187,
  253,
  94,
  19,
  63,
  20,
  203,
  53,
  92,
  63,
  216,
  217,
  147,
  103,
  150,
  79,
]
`;

exports[`Hash > keccak_256 2`] = `
Uint8Array [
  172,
  175,
  50,
  137,
  215,
  182,
  1,
  203,
  209,
  20,
  251,
  54,
  196,
  210,
  156,
  133,
  187,
  253,
  94,
  19,
  63,
  20,
  203,
  53,
  92,
  63,
  216,
  217,
  147,
  103,
  150,
  79,
]
`;

exports[`Hash > keccak_384 1`] = `
Uint8Array [
  77,
  96,
  137,
  47,
  222,
  127,
  150,
  123,
  202,
  189,
  196,
  124,
  115,
  18,
  42,
  230,
  49,
  31,
  161,
  249,
  190,
  144,
  215,
  33,
  218,
  50,
  3,
  15,
  116,
  103,
  162,
  227,
  219,
  63,
  156,
  203,
  60,
  116,
  100,
  131,
  249,
  210,
  184,
  118,
  227,
  157,
  239,
  23,
]
`;

exports[`Hash > keccak_384 2`] = `
Uint8Array [
  77,
  96,
  137,
  47,
  222,
  127,
  150,
  123,
  202,
  189,
  196,
  124,
  115,
  18,
  42,
  230,
  49,
  31,
  161,
  249,
  190,
  144,
  215,
  33,
  218,
  50,
  3,
  15,
  116,
  103,
  162,
  227,
  219,
  63,
  156,
  203,
  60,
  116,
  100,
  131,
  249,
  210,
  184,
  118,
  227,
  157,
  239,
  23,
]
`;

exports[`Hash > keccak_512 1`] = `
Uint8Array [
  237,
  167,
  101,
  87,
  108,
  132,
  198,
  0,
  237,
  127,
  93,
  151,
  81,
  14,
  146,
  112,
  59,
  97,
  245,
  33,
  93,
  239,
  42,
  22,
  16,
  55,
  253,
  157,
  209,
  245,
  182,
  237,
  79,
  134,
  206,
  70,
  7,
  60,
  14,
  63,
  52,
  181,
  45,
  224,
  40,
  158,
  156,
  97,
  135,
  152,
  255,
  249,
  221,
  75,
  27,
  254,
  3,
  91,
  219,
  134,
  69,
  252,
  110,
  55,
]
`;

exports[`Hash > keccak_512 2`] = `
Uint8Array [
  237,
  167,
  101,
  87,
  108,
  132,
  198,
  0,
  237,
  127,
  93,
  151,
  81,
  14,
  146,
  112,
  59,
  97,
  245,
  33,
  93,
  239,
  42,
  22,
  16,
  55,
  253,
  157,
  209,
  245,
  182,
  237,
  79,
  134,
  206,
  70,
  7,
  60,
  14,
  63,
  52,
  181,
  45,
  224,
  40,
  158,
  156,
  97,
  135,
  152,
  255,
  249,
  221,
  75,
  27,
  254,
  3,
  91,
  219,
  134,
  69,
  252,
  110,
  55,
]
`;

exports[`Hash > ripemd160 1`] = `
Uint8Array [
  82,
  122,
  106,
  75,
  154,
  109,
  167,
  86,
  7,
  84,
  104,
  66,
  224,
  224,
  1,
  5,
  53,
  11,
  26,
  175,
]
`;

exports[`Hash > ripemd160 2`] = `
Uint8Array [
  82,
  122,
  106,
  75,
  154,
  109,
  167,
  86,
  7,
  84,
  104,
  66,
  224,
  224,
  1,
  5,
  53,
  11,
  26,
  175,
]
`;

exports[`Hash > sha1 1`] = `
Uint8Array [
  10,
  10,
  159,
  42,
  103,
  114,
  148,
  37,
  87,
  171,
  83,
  85,
  215,
  106,
  244,
  66,
  248,
  246,
  94,
  1,
]
`;

exports[`Hash > sha1 2`] = `
Uint8Array [
  10,
  10,
  159,
  42,
  103,
  114,
  148,
  37,
  87,
  171,
  83,
  85,
  215,
  106,
  244,
  66,
  248,
  246,
  94,
  1,
]
`;

exports[`Hash > sha3_224 1`] = `
Uint8Array [
  133,
  48,
  72,
  251,
  139,
  17,
  70,
  43,
  97,
  0,
  56,
  86,
  51,
  192,
  204,
  141,
  205,
  198,
  226,
  184,
  227,
  118,
  194,
  129,
  2,
  188,
  132,
  242,
]
`;

exports[`Hash > sha3_224 2`] = `
Uint8Array [
  133,
  48,
  72,
  251,
  139,
  17,
  70,
  43,
  97,
  0,
  56,
  86,
  51,
  192,
  204,
  141,
  205,
  198,
  226,
  184,
  227,
  118,
  194,
  129,
  2,
  188,
  132,
  242,
]
`;

exports[`Hash > sha3_256 1`] = `
Uint8Array [
  26,
  241,
  122,
  102,
  78,
  63,
  168,
  228,
  25,
  184,
  186,
  5,
  194,
  161,
  115,
  22,
  157,
  247,
  97,
  98,
  165,
  162,
  134,
  224,
  196,
  5,
  180,
  96,
  212,
  120,
  247,
  239,
]
`;

exports[`Hash > sha3_256 2`] = `
Uint8Array [
  26,
  241,
  122,
  102,
  78,
  63,
  168,
  228,
  25,
  184,
  186,
  5,
  194,
  161,
  115,
  22,
  157,
  247,
  97,
  98,
  165,
  162,
  134,
  224,
  196,
  5,
  180,
  96,
  212,
  120,
  247,
  239,
]
`;

exports[`Hash > sha3_384 1`] = `
Uint8Array [
  170,
  154,
  216,
  164,
  159,
  49,
  210,
  221,
  202,
  187,
  183,
  1,
  10,
  21,
  102,
  65,
  124,
  255,
  128,
  63,
  239,
  80,
  235,
  162,
  57,
  85,
  136,
  38,
  248,
  114,
  228,
  104,
  197,
  116,
  62,
  127,
  2,
  107,
  10,
  142,
  91,
  45,
  122,
  28,
  196,
  101,
  205,
  190,
]
`;

exports[`Hash > sha3_384 2`] = `
Uint8Array [
  170,
  154,
  216,
  164,
  159,
  49,
  210,
  221,
  202,
  187,
  183,
  1,
  10,
  21,
  102,
  65,
  124,
  255,
  128,
  63,
  239,
  80,
  235,
  162,
  57,
  85,
  136,
  38,
  248,
  114,
  228,
  104,
  197,
  116,
  62,
  127,
  2,
  107,
  10,
  142,
  91,
  45,
  122,
  28,
  196,
  101,
  205,
  190,
]
`;

exports[`Hash > sha3_512 1`] = `
Uint8Array [
  56,
  224,
  92,
  51,
  215,
  176,
  103,
  18,
  127,
  33,
  125,
  140,
  133,
  110,
  85,
  79,
  207,
  240,
  156,
  147,
  32,
  184,
  165,
  151,
  156,
  226,
  255,
  93,
  149,
  221,
  39,
  186,
  53,
  209,
  251,
  165,
  12,
  86,
  45,
  253,
  29,
  108,
  196,
  139,
  201,
  197,
  186,
  164,
  57,
  8,
  148,
  65,
  140,
  201,
  66,
  217,
  104,
  249,
  123,
  203,
  101,
  148,
  25,
  237,
]
`;

exports[`Hash > sha3_512 2`] = `
Uint8Array [
  56,
  224,
  92,
  51,
  215,
  176,
  103,
  18,
  127,
  33,
  125,
  140,
  133,
  110,
  85,
  79,
  207,
  240,
  156,
  147,
  32,
  184,
  165,
  151,
  156,
  226,
  255,
  93,
  149,
  221,
  39,
  186,
  53,
  209,
  251,
  165,
  12,
  86,
  45,
  253,
  29,
  108,
  196,
  139,
  201,
  197,
  186,
  164,
  57,
  8,
  148,
  65,
  140,
  201,
  66,
  217,
  104,
  249,
  123,
  203,
  101,
  148,
  25,
  237,
]
`;

exports[`Hash > sha256 1`] = `
Uint8Array [
  223,
  253,
  96,
  33,
  187,
  43,
  213,
  176,
  175,
  103,
  98,
  144,
  128,
  158,
  195,
  165,
  49,
  145,
  221,
  129,
  199,
  247,
  10,
  75,
  40,
  104,
  138,
  54,
  33,
  130,
  152,
  111,
]
`;

exports[`Hash > sha256 2`] = `
Uint8Array [
  223,
  253,
  96,
  33,
  187,
  43,
  213,
  176,
  175,
  103,
  98,
  144,
  128,
  158,
  195,
  165,
  49,
  145,
  221,
  129,
  199,
  247,
  10,
  75,
  40,
  104,
  138,
  54,
  33,
  130,
  152,
  111,
]
`;

exports[`Hash > sha384 1`] = `
Uint8Array [
  84,
  133,
  204,
  155,
  51,
  101,
  180,
  48,
  93,
  251,
  78,
  131,
  55,
  224,
  165,
  152,
  165,
  116,
  248,
  36,
  43,
  241,
  114,
  137,
  224,
  221,
  108,
  32,
  163,
  205,
  68,
  160,
  137,
  222,
  22,
  171,
  74,
  179,
  8,
  246,
  62,
  68,
  177,
  23,
  14,
  181,
  245,
  21,
]
`;

exports[`Hash > sha384 2`] = `
Uint8Array [
  84,
  133,
  204,
  155,
  51,
  101,
  180,
  48,
  93,
  251,
  78,
  131,
  55,
  224,
  165,
  152,
  165,
  116,
  248,
  36,
  43,
  241,
  114,
  137,
  224,
  221,
  108,
  32,
  163,
  205,
  68,
  160,
  137,
  222,
  22,
  171,
  74,
  179,
  8,
  246,
  62,
  68,
  177,
  23,
  14,
  181,
  245,
  21,
]
`;

exports[`Hash > sha512 1`] = `
Uint8Array [
  55,
  77,
  121,
  74,
  149,
  205,
  207,
  216,
  179,
  89,
  147,
  24,
  95,
  239,
  155,
  163,
  104,
  241,
  96,
  216,
  218,
  244,
  50,
  208,
  139,
  169,
  241,
  237,
  30,
  90,
  190,
  108,
  198,
  146,
  145,
  224,
  250,
  47,
  224,
  0,
  106,
  82,
  87,
  14,
  241,
  140,
  25,
  222,
  244,
  230,
  23,
  195,
  60,
  229,
  46,
  240,
  166,
  229,
  251,
  227,
  24,
  203,
  3,
  135,
]
`;

exports[`Hash > sha512 2`] = `
Uint8Array [
  55,
  77,
  121,
  74,
  149,
  205,
  207,
  216,
  179,
  89,
  147,
  24,
  95,
  239,
  155,
  163,
  104,
  241,
  96,
  216,
  218,
  244,
  50,
  208,
  139,
  169,
  241,
  237,
  30,
  90,
  190,
  108,
  198,
  146,
  145,
  224,
  250,
  47,
  224,
  0,
  106,
  82,
  87,
  14,
  241,
  140,
  25,
  222,
  244,
  230,
  23,
  195,
  60,
  229,
  46,
  240,
  166,
  229,
  251,
  227,
  24,
  203,
  3,
  135,
]
`;

exports[`Hash > shake128 1`] = `
Uint8Array [
  43,
  245,
  230,
  222,
  230,
  7,
  159,
  173,
  96,
  79,
  87,
  49,
  148,
  186,
  132,
  38,
]
`;

exports[`Hash > shake128 2`] = `
Uint8Array [
  43,
  245,
  230,
  222,
  230,
  7,
  159,
  173,
  96,
  79,
  87,
  49,
  148,
  186,
  132,
  38,
]
`;

exports[`Hash > shake256 1`] = `
Uint8Array [
  179,
  190,
  151,
  191,
  217,
  120,
  131,
  58,
  101,
  88,
  140,
  234,
  232,
  163,
  76,
  245,
  158,
  149,
  88,
  90,
  246,
  32,
  99,
  230,
  184,
  157,
  7,
  137,
  243,
  114,
  66,
  78,
]
`;

exports[`Hash > shake256 2`] = `
Uint8Array [
  179,
  190,
  151,
  191,
  217,
  120,
  131,
  58,
  101,
  88,
  140,
  234,
  232,
  163,
  76,
  245,
  158,
  149,
  88,
  90,
  246,
  32,
  99,
  230,
  184,
  157,
  7,
  137,
  243,
  114,
  66,
  78,
]
`;
